settings.gradle.kts

```diff
@@ -64,3 +64,4 @@ include(":rag:vector-storage")
 include(":koog-spring-boot-starter")
 
 include(":koog-ktor")
+include(":docs")
```

Comment:
You can do it as a [composite build](https://docs.gradle.org/current/userguide/composite_builds.html) to separate it from the build of other modules

Reply:
I've tried, but there are some issues with `libs.versions.toml` as if we build it as a composite project, it can not access it(